idaes/config.py

```diff
@@ -19,67 +19,62 @@
 
 _log = logging.getLogger(__name__)
 # Default release version if no options provided for get-extensions
-default_binary_release = "2.5.6"
+default_binary_release = "2.6.0"
 # Where to download releases from get-extensions
 release_base_url = "https://github.com/IDAES/idaes-ext/releases/download"
 # Where to get release checksums
 release_checksum_url = \
     "https://raw.githubusercontent.com/IDAES/idaes-ext/main/releases/sha256sum_{}.txt"
 # Map some platform names to others for get-extensions
 binary_platform_map = {
-    "rhel7": "centos7",
-    "rhel8": "centos8",
-    "ubuntu1810": "ubuntu1804",
-    "ubuntu1904": "ubuntu1804",
-    "ubuntu1910": "ubuntu1804",
-    "ubuntu2010": "ubuntu2004",
-    "ubuntu2104": "ubuntu2004",
-    "ubuntu2110": "ubuntu2004",
-    "linux": "centos7",
+    "rhel7": "el7",
+    "rhel8": "el8",
+    "centos7": "el7",
+    "centos8": "el8",
+    "rocky8": "el8",
+    "almalinux8": "el8",
+    "debian9": "el7",
+    "debian10": "el8",
+    "debian11": "ubuntu2004",
+    "ubuntu1804": "ubuntu1804",
+    "ubuntu2004": "ubuntu2004",
+    "ubuntu2204": "ubuntu2204",
+    "kubuntu1804": "kubuntu1804",
+    "kubuntu2004": "kubuntu2004",
+    "kubuntu2204": "kubuntu2204",
+    "xubuntu1804": "xubuntu1804",
+    "xubuntu2004": "xubuntu2004",
+    "xubuntu2204": "xubuntu2204",
 }
-# Set of known platforms with available binaries and descriptions of them
-known_binary_platform = {
-    "auto":"Auto-select windows, darwin or linux",
-    "windows":"Microsoft Windows (built on verion 20H2 with MinGW)",
-    "darwin": "OSX (currently not available)",
-    "linux": (
-            f"Linux (auto select distribution or fall back on "
-            f"{binary_platform_map['linux']})"
-        ),
-    "centos7": "CentOS 7",
-    "centos8": "CentOS 8",
-    "rhel7": "Red Hat Enterprise Linux 7",
-    "rhel8": "Red Hat Enterprise Linux 8",
-    "ubuntu1804": "Ubuntu 18.04",
-    "ubuntu1810": "Ubuntu 18.10",
-    "ubuntu1904": "Ubuntu 19.04",
-    "ubuntu1910": "Ubuntu 19.10",
-    "ubuntu2004": "Ubuntu 20.04",
-    "ubuntu2010": "Ubuntu 20.10",
-    "ubuntu2104": "Ubuntu 21.04",
+# Machine map
+binary_machine_map = {
```

Comment:
Again not important, but looking at this I wonder if it could be made more compact and robust by using Pythons string capitalization methods.

Reply:
I thought about that.  I think I caught the reasonable possibilities.  This lets me just use a dictionary without worrying about calling a function (not that it's really a big deal).